lib/exercism/team.rb

```diff
@@ -74,9 +74,13 @@ def invite(users, inviter)
     return unless users.present?
 
     users = Array(users) - all_members
-
-    users.each do |user|
-      TeamMembershipInvite.create(user: user, team: self, inviter: inviter, refused: false)
+    self.transaction do
+      users.map do |user|
```

Comment:
ohhh... I left the map but forgot to comment and do the other change,I was mean to discuss if you are agree to change this for the sake of 'better' (easy?) test and other future uses of the method. I use the mapping on multiple `create` on my own projects, but not sure if you will adopt.
Currently the method `invite` returns the users collection. That's something not usable in any other way, with this change the method will return a collection of invitations just created, ergo the System Under Test will have something to assert right there after their execution, not needing any other query, or future verification. It's isolated and can resist parallel testing because it will run on their own. I know it's YAGNI, but pretty sure it will not harm the current state (and perhaps it's only my C background talking 😃 ).

Before...
```ruby
  def test_team_invite
    inviter = User.create(username: 'inviter')

    team = Team.by(alice).defined_with(slug: 'blurple')
    team.save

    team.invite(bob, inviter)
    team.reload

    refute team.includes?(bob)
    assert team.member_invites.include?(bob)
  end
```

After

```ruby
  def test_team_invite
    inviter = User.create(username: 'inviter')
    candidates = [User.create(username: 'McHammer'), User.create(user: 'VanillaIce')]
    team = Team.by(alice).defined_with(slug: 'blurple')
    team.save

    invitations = team.invite(candidates, inviter)
    # Assertions against invitations, perhaps invite should return a
    # composed object TeamMembershipInvite + User.
    assert_equal 2, invitations.size
  end
```